napari/utils/events/event.py

```diff
@@ -594,7 +594,16 @@ def __call__(self, *args, **kwargs) -> Event:
     def _invoke_callback(self, cb: Callback, event: Event):
         try:
             cb(event)
-        except Exception:
+        except Exception as e:
+            # dead Qt object with living python pointer. not importing Qt
+            # here... but this error is consistent across backends
+            if (
+                isinstance(e, RuntimeError)
+                and str(e).startswith('wrapped')
+                and 'has been deleted' in str(e)
+            ):
+                self.disconnect(cb)
```

Comment:
Is there a chance this disconnect errors?

Reply:
seems _extremely_ unlikely.  the `cb` variable in this case has just been pulled from `self._callbacks` and all disconnect does is:
```py
        if cb is None:
            self._callbacks = []
            self._callback_refs = []
        else:
            cb = self._normalize_cb(cb)
            if cb in self._callbacks:
                idx = self._callbacks.index(cb)
                self._callbacks.pop(idx)
                self._callback_refs.pop(idx)
```

and `_normalize(cb)` in this case will just return `cb`